src/Spectre.Algorithms/Results/DivikResult.cs

```diff
@@ -216,18 +231,24 @@ private DivikResult[] ParseSubregions(MWStructArray array)
             }
             return null;
         }
+
         #endregion
 
         #region Save
+
         /// <summary>
         /// Saves result under the specified path in JSON format.
         /// </summary>
         /// <param name="path">The destination path.</param>
-        /// <exception cref="System.NotImplementedException">Always, as result saving has not been implemented yet.</exception>
         public void Save(string path)
         {
-            throw new NotImplementedException("Result saving has not been implemented yet.");
+            string data = JsonConvert.SerializeObject(this);
+            using (StreamWriter sw = File.CreateText(path))
```

Comment:
File.WriteAllText would certainly fit better here. This is actually something that I've been looking for, but decided to pick StreamWriter because of lack of time and as that was the first thing that I came across. 
There indeed are exceptions that are thrown by both of these, and I was truly wondering whether I should list all of them here, since there are a plenty of them and their descriptions would just reproduce those found in CreateText method.